index.d.ts

```diff
@@ -349,7 +349,7 @@ declare module "ldclient-node" {
      */
     variation: (key: string, user: LDUser, defaultValue: LDFlagValue, callback?: (err: any, res: LDFlagValue) => void) => void;
 
-    toggle: typeof variation;
+    toggle: (key: string, user: LDUser, defaultValue: LDFlagValue, callback?: (err: any, res: LDFlagValue) => void) => void;
```

Comment:
I don't know of a terse way to replicate a function type definition. If `toggle` is deprecated, maybe the solution is to just remove it from the type definitions altogether?